tests/calculations/test_example.py

```diff
@@ -37,14 +35,10 @@ def _generate_calc_job(folder, builder):
 
 
 def load_module(module_name, full_path):
-    if six.PY2:
-        import imp
-        module = imp.load_source(module_name, full_path)
-    else:
-        import importlib.util  # pylint: disable=import-error
-        spec = importlib.util.spec_from_file_location(module_name, full_path)
-        module = importlib.util.module_from_spec(spec)
-        spec.loader.exec_module(module)
+    import importlib.util  # pylint: disable=import-error
```

Comment:
Probably we don't need this disable anymore (this was to avoid that the py2 pylint complained)

Reply:
True! Funny that pylint didn't throw a `useless-suppression`, though.

Reply:
interesting - I never saw this message!